routing/turns.cpp

```diff
@@ -52,6 +52,20 @@ static_assert(g_turnNames.size() == static_cast<size_t>(TurnDirection::Count),
 namespace routing
 {
 // UniNodeId -------------------------------------------------------------------
+std::atomic<uint32_t> UniNodeId::m_nextFakeId(0);
```

Comment:
Done. But m_nextFakeId is set to osrm NodeID field. The type of the field is unsigned int